activation/2.0/_index.md

```diff
@@ -1,35 +1,31 @@
 ---
-title: "Jakarta Activation 2.0 (under development)"
-date: 2020-mm-dd
+title: "Jakarta Activation 2.0"
+date: 2020-06-05
 summary: "Release for Jakarta EE 9"
 ---
-# Need scope statement
+Jakarta Activation defines a set of standard services to: determine the
+MIME type of an arbitrary piece of data; encapsulate access to it;
+discover the operations available on it; and instantiate the
+appropriate bean to perform the operation(s).
 
-* [Jakarta Activation 2.0 Release Record]()
+* [Jakarta Activation 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jaf/releases/2.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Activation 2.0 Specification Document]() (PDF)
-* [Jakarta Activation 2.0 Specification Document]() (HTML)
+* [Jakarta Activation 2.0 Specification Document](activation-spec-2.0.pdf) (PDF)
+* [Jakarta Activation 2.0 Specification Document](activation-spec-2.0.html) (HTML)
 * [Jakarta Activation 2.0 Javadoc](./apidocs)
-* [Jakarta Activation 2.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Activation 2.0 TCK](http://downloads.eclipse.org/jakarta/activation/2.0/jakarta-activation-tck-2.0.0.zip) ([sig](http://downloads.eclipse.org/jakarta/activation/2.0/jakarta-activation-tck-2.0.0.zip.sig),[sha](http://downloads.eclipse.org/jakarta/activation/2.0/jakarta-activation-tck-2.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-  * [jakarta.activation:jakarta.activation-api:jar:2.0.0]()
+  * [jakarta.activation:jakarta.activation-api:jar:2.0.0](https://search.maven.org/artifact/jakarta.activation/jakarta.activation-api/2.0.0/jar)
```

Comment:
RC3 is available in Maven, but not the final 2.0 release...
https://search.maven.org/artifact/jakarta.activation/jakarta.activation-api/2.0.0-RC3/jar

Reply:
final release cannot be published in central without ballot, correct? The thing is, that 2.0.0 FINAL is available in staging since April 16, see https://jakarta.oss.sonatype.org/content/groups/staging/jakarta/activation/jakarta.activation-api/2.0.0/ (best way to search is to go through the search at https://jakarta.oss.sonatype.org/ - this way one can easily find out what is in central and what is in staging), so final artifacts created by Bill are going to disappear around June 11 I guess (after 60days, correct?). Repository moved already. If there is a chance to avoid rebuilding final bits, I prefer to go for it

Reply:
@lukasj Thanks.  I know about the staging area.  I just don't think we have the time to do a final release of Activation before the Milestone release.  But, maybe that's okay.  Maybe we just don't publicize the availability of independent Activation Specification at this time.  Maybe you should just start the formal process of releasing Activation 2.0, which require all of the necessary "final" bits and initiate a ballot.  If that's the direction you are choosing, then many of my earlier comments no longer apply.  So, let's figure out your direction for the Activation release and move on from there.  Thanks!

Reply:
I thought that we are putting there URL of the artifact which will be available in Maven Central *after* ballot is passed. In this case it's correct.
IMO, I don't like the requirement of adding non-existing URLs. It's easy to make a mistake. I understand that the only reason of adding it is that the PR is not updated after the corresponding artifacts get released to Maven Central. I think it's better to provide staging links for the ballot and update the PR with public links when the artifacts get publicly released.

Reply:
Old conversation, resolving...